src/Controls/tests/DeviceTests/Elements/CollectionView/CollectionViewTests.cs

```diff
@@ -143,7 +143,7 @@ public async Task CollectionViewCanSizeToContent(CollectionViewSizingTestCase te
 					await WaitForUIUpdate(frame, collectionView);
 					frame = collectionView.Frame;
 
-#if WINDOWS
+#if WINDOWS || ANDROID
```

Comment:
This PR adds a slightly longer delay to [`OnLoaded`](https://github.com/dotnet/maui/pull/16046/files) this gives Android a little more time to run an `OnMeasure` loop. Because of this, `Android` also needs this wait.